tests/release/release.go

```diff
@@ -50,12 +50,12 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1108]test-release-service-happy-p
 
 		GinkgoWriter.Println("Managed Namespace :", managedNamespace)
 
-		// Wait until the "pipeline" SA is created and ready with secrets by the openshift-pipelines operator
-		GinkgoWriter.Printf("Wait until the '%s' SA is created in %s namespace \n", serviceAccount, managedNamespace)
-		Eventually(func() bool {
-			sa, err := fw.AsKubeAdmin.CommonController.GetServiceAccount(serviceAccount, managedNamespace)
-			return sa != nil && err == nil
-		}, pipelineServiceAccountCreationTimeout, defaultInterval).Should(BeTrue(), "timed out when waiting for the \"pipeline\" SA to be created")
+		// // Wait until the "pipeline" SA is created and ready with secrets by the openshift-pipelines operator
```

Comment:
Why is commented?

Reply:
it's not used anymore, 
thanks for reminding me, I remove it .